### DISCUSSION TITLE
Unauthorized - no public worker access

### DISCUSSION THREAD

**Discussion by tylerpitchford** [Budibase/budibase#1832] [Help]
I'm attempting to run Budibase but after setting up an admin account and attempting to login, I'm getting:

- Unauthorized - no public worker access

I've tried both docker-compose directly and the CLI tool, same issue on both. The UI flashes that the login worked, the network log shows /analytics succeeds, /checklist returns:

`{"apps":0,"smtp":false,"adminUser":true,"oauth":false}`

but the call to /self throws:

- 403, Unauthorized - no public worker access

Cheers,
Tyler

**Comment by shogunpurple** [comment]
Hi @tylerpitchford - thanks for the report. This is quite an odd one. Could you try this in incognito or clear your browser cookies before logging in? Does it still happen?

**Reply by tylerpitchford** [reply]
Ran a login attempt in an private window in Firefox with the same result. That said, I was able to get logged in finally by switching to Chrome I guess it's a Firefox incompatibility. 

Thanks for the quick reply, much appreciated!

**Reply by shogunpurple** [reply]
@tylerpitchford I use firefox too but haven't come across that one - will look into it. Thank you!

**Reply by tylerpitchford** [reply]
If it's any help, I'm on Windows 10, running Budibase from Docker/WSL2 and my Firefox user-agent is:

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0

No extensions installed.

**Reply by shogunpurple** [reply]
Hey @tylerpitchford - I could not recreate this one at all, even on a windows VM in FF. I feel like it's possibly an environmental/cookie issue, Is it still happening to you?
